pkg/azurefile/controllerserver.go

```diff
@@ -104,6 +106,11 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 	}
 
 	if acquired := d.volumeLocks.TryAcquire(volName); !acquired {
+		// logging the job status if it's volume cloning
+		if req.GetVolumeContentSource() != nil {
+			jobState, percent, err := getAzcopyJob(volName)
+			klog.V(2).Infof("azcopy job status: %s, copy percent: %s%, error: %v", jobState, percent, err)
```

Comment:
`copy percent: %.1f%`

Reply:
`percent` is a string parsed by azcopy and it is already %1.f format.

Reply:
use copy percent: %s%% to fix print nil error